### DISCUSSION TITLE
Warnings when uploading apk to google play

### DISCUSSION THREAD

**Discussion by shaymolho** [airsdk/Adobe-Runtime-Support#1346] [Q&A] [answered]
When I upload I have these warnings:

- There is no deobfuscation file associated with this APK. If you use obfuscated code (R8/proguard), uploading a deobfuscation file will make crashes and ANRs easier to analyze and debug. Using R8/proguard can help reduce app size. [Learn More](https://developer.android.com/studio/build/shrink-code#decode-stack-trace)
- This APK contains native code, and you've not uploaded debug symbols. We recommend you upload a symbol file to make your crashes and ANRs easier to analyze and debug. [Learn More](https://developer.android.com/studio/build/shrink-code#native-crash-support)

How to resolve these warnings? @ajwfrost your help is appreciated
Using latest AIR (633) and uploading APKs (not AAB)

**Comment by ajwfrost** [comment, marked as answer]
You can ignore those ... we can look into the proguard part within the context of the Gradle-based compilations, but for anything other than the ANEs, these would be referring to the AIR source code itself which we're not able to provide debug symbols for. We have the symbol files here and if you provide a crash report that has just the function addresses, we can interpret that using the .gdb file here...

thanks
